README.md

```diff
@@ -1,3 +1,132 @@
 # repo-template
 
 **This is not an official Google product.**
+
+This template is for creating an AOD (access on demain) repository for GCP organizations, folders, and projects.
+
+After completing the installation. Follow below steps to file an AOD request.
+1. Open a PR
+  - A title and description to provide enough justification. E.g. "AOD request: debug customer issue X".
+  - Add following file at the repo root, note that any changes with below file changes will be considered AOD request and thus cannot be merged:
+    - iam.yaml - Request for IAM permissions on organization / folder / project level. To see an example of this file, refer to the example-iam.yaml file.
+  - Optionally use predefined duration labels on the PR to specify the IAM permission expiration. The duration is default to 2h if no label is used.
+
+2. PR approval
+  - Github workflows will check if the PR is an AOD request, marked by a "do_not_merge" failure workflow, and if it is a valid AOD request.
+  - Upon PR approval, the request will be handled by AOD.
+
+3. Close the PR after the request is handled successfully.
+
+## Installation
+
```

Comment:
Define "who" should be carrying out these steps - likely someone with admin role on the project/folder/org